### PR TITLE
added 5.1.1 and 5.1.2 for containerregistry

### PR DESCRIPTION
**Pull request by KonradSchieban**

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -47,4 +47,42 @@ inputs:
...
+  type: array
+  value:
+    - "serviceAccount:test1@myproject.iam.gserviceaccount.com"
+    - "serviceAccount:test2@myproject.iam.gserviceaccount.com"
+- name: registry_storage_object_admin_list
+  description: 'The allowed list of Storage Object Admins on Registry image bucket'
+  type: array
+  value:
+    - "serviceAccount:test1@myproject.iam.gserviceaccount.com"
+    - "serviceAccount:test2@myproject.iam.gserviceaccount.com"
```
[the remark this answers is not in the document]
Comment (KonradSchieban) [reply inspec.yml:62]: The scanner will definitely complain.
My inputs.yml overwrites these arrays with [].

[the remark this answers is not in the document]
Comment (KonradSchieban) [reply inspec.yml]: Addressed.

File: controls/5.01-container-registry.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  tag cis_version: cis_version.to_s
+  tag project: gcp_project_id.to_s
+
+  ref 'CIS Benchmark', url: cis_url.to_s
+  ref 'GCP Docs', url: 'https://cloud.google.com/container-registry/docs/access-control'
+
+  # Check if storage bucket exists
+  if google_storage_bucket(name: "artifacts.#{gcp_project_id}.appspot.com").name.nil?
+    impact 'none'
+    describe "[#{gcp_project_id}] does not have a storage bucket for Google Container Registry Images, this test is Not Applicable.." do
```
Comment (binamov) [inline controls/5.01-container-registry.rb:88]: two dots at the end of the string

Comment (KonradSchieban) [reply controls/5.01-container-registry.rb]: Fixed.

**Review by binamov — COMMENTED** [review]
🍦 🍰 🍵

**Pull request merged**